Card/card.css

```diff
@@ -0,0 +1,151 @@
+@import url("../../Utility CSS/navbar.css");
+
+.box {
+  display: flex;
+  flex-direction: row;
```

Comment:
There's no need to mention flex-direction : row , it's row by default

Reply:
done